core/src/main/java/uk/ac/bbsrc/tgac/miso/core/util/FormUtils.java

```diff
@@ -2997,164 +2997,6 @@ public static JSONArray preProcessSampleSheetImport(File inPath, User u, Request
     }
   }
 
-  public static JSONArray processSampleSheetImport(File inPath, User u, RequestManager manager) throws Exception {
```

Comment:
Not used from anywhere on develop branch. Confirmed now with @frankie13 that their internal config branch doesn't use it either. I had previously assumed that the "config" branch doesn't do forms processing stuff, but better safe I suppose!